octobot_trading/enums.py

```diff
@@ -61,6 +61,10 @@ class OrderStatus(enum.Enum):
     EXPIRED = "expired"
     REJECTED = "rejected"
     UNKNOWN = "unknown"
+    ORDER_CANCELED = "ORDER_CANCELED"
+    ORDER_FILLED = "ORDER_FILLED"
+    PARTIALLY_FILLED_CANCELED = "PARTIALLY_FILLED_CANCELED"
+    ORDER_NEW = "ORDER_NEW"
```

Comment:
What's the `ORDER_NEW` status?

Reply:
I removed it and moved it to the generic orders parser STATUS_MAP

Reply:
Its a status that bybit spot uses